### ISSUE TITLE
ToDo's (II)

### ISSUE THREAD

**Issue by JackMD** [JackMD/MysteryCrate#67]
Mention the stuff you want to be added and I will try my best to add it.
This is continued after #34 .

**Comment by ZandercraftGames** [comment]
Fix errors when giving a custom-enchanted item.

**Comment by ZandercraftGames** [comment]
Allow multiple crates to be opened at once.

**Comment by JackMD** [comment]
@ZandercraftGames what errors are you talking about? There are no errors for or unless you messed up you crates.yml configuration which I am 101% sure you did.

**Comment by JackMD** [comment]
Also multiple crate opening is not happening

**Comment by ZandercraftGames** [comment]
@JackMD `[15:44:03] [Server thread/CRITICAL]: [MysteryCrate] Could not execute task JackMD\MysteryCrate\UpdaterEvent: Undefined index: level
[15:44:03] [Server thread/CRITICAL]: ErrorException: "Undefined index: level" (EXCEPTION) in "MysteryCrate.phar/src/JackMD/MysteryCrate/UpdaterEvent" at line 100
[15:44:03] [Server thread/DEBUG]: #0 MysteryCrate.phar/src/JackMD/MysteryCrate/UpdaterEvent(100): pocketmine\utils\Utils::errorExceptionHandler(integer 8, string Undefined index: level, string phar://C:/Users/Zander Maitland/Desktop/Pocketmine/plugins/MysteryCrate.phar/src/JackMD/MysteryCrate/UpdaterEvent.php, integer 100, array Array())
[15:44:03] [Server thread/DEBUG]: #1 src/pocketmine/scheduler/TaskHandler(163): JackMD\MysteryCrate\UpdaterEvent->onRun(integer 96149)
[15:44:03] [Server thread/DEBUG]: #2 src/pocketmine/scheduler/TaskScheduler(203): pocketmine\scheduler\TaskHandler->run(integer 96149)
[15:44:03] [Server thread/DEBUG]: #3 src/pocketmine/plugin/PluginManager(711): pocketmine\scheduler\TaskScheduler->mainThreadHeartbeat(integer 96149)
[15:44:03] [Server thread/DEBUG]: #4 src/pocketmine/Server(2509): pocketmine\plugin\PluginManager->tickSchedulers(integer 96149)
[15:44:03] [Server thread/DEBUG]: #5 src/pocketmine/Server(2258): pocketmine\Server->tick()
[15:44:03] [Server thread/DEBUG]: #6 src/pocketmine/Server(2132): pocketmine\Server->tickProcessor()
[15:44:03] [Server thread/DEBUG]: #7 src/pocketmine/Server(1713): pocketmine\Server->start()
[15:44:03] [Server thread/DEBUG]: #8 src/pocketmine/PocketMine(237): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string C:\Users\Zander Maitland\Desktop\Pocketmine\, string C:\Users\Zander Maitland\Desktop\Pocketmine\plugins\)
[15:44:03] [Server thread/DEBUG]: #9 C:/Users/Zander Maitland/Desktop/Pocketmine/PocketMine-MP.phar(1): require(string phar://C:/Users/Zander Maitland/Desktop/Pocketmine/PocketMine-MP.phar/src/pocketmine/PocketMine.php)`

**Comment by ZandercraftGames** [comment]
Also... about multi-crate opening... The issue is that even your code for making only 1 crate able to be opened at once is flawed. If someone is opening the mythic crate, and while they are doing that someone tries to open the legendary crate, then the items from the mythic crate stop circling in the gui and the legendary crates contents start circling. Then the person who opened the Mythic crate, gets the Legendary crate's contents

**Comment by JackMD** [comment]
Please open a new issue and this be a todo section
Also send your crates.yml in the new issue
